blocks/library/image/block.js

```diff
@@ -220,10 +245,61 @@ class ImageBlock extends Component {
 								onChange={ this.updateImageURL }
 							/>
 						) }
+						<div className="blocks-image__dimensions">
+							<p className="blocks-image__dimensions__row">
+								{ __( 'Image Dimensions' ) }
+							</p>
+							<div className="blocks-image__dimensions__row">
+								<TextControl
+									type="number"
+									className="blocks-image__dimensions__width"
+									label={ __( 'Width' ) }
+									value={ width !== undefined ? width : '' }
+									placeholder={ selectedSize.width }
```

Comment:
This can apparently be unassigned, resulting in warnings logged to the console.

Issue at #8026.